packages/typespec-vscode/package.json

```diff
@@ -59,13 +59,24 @@
     ],
     "configuration": [
       {
-        "title": "TypeSpec Language Server Path",
+        "title": "TypeSpec",
         "properties": {
           "typespec.tsp-server.path": {
             "type": "string",
             "default": "",
             "description": "Path to `tsp-server` command that runs the TypeSpec language server.\n\nIf not specified, then `tsp-server` found on PATH is used.\n\nExample (User): /usr/local/bin/tsp-server\nExample (Workspace): ${workspaceFolder}/node_modules/@typespec/compiler",
             "scope": "machine-overridable"
+          },
+          "typespecLanguageServer.trace.server": {
+            "scope": "window",
+            "type": "string",
+            "enum": [
+              "off",
+              "messages",
```

Comment:
Shouldn't that be inline with the log level?

Reply:
No, this is a config just for trace log, when a trace is logged by method ```trace(msg, verbose)```, "off" will trace nothing, "messages" will only trace the 'msg', and "verbose" will trace both the msg and verbose. 

In case if you are interested, the config actually has 4 values: off, messages, compact and verbose. The 'compact' would compact the verbose when stringify it into json which is not quite useful to us. and I also checked a few other extensions and they don't include the 'compat' either [like this](https://github.com/microsoft/vscode/blob/d40dff9ef9aacb8e5226bcf3938c71f4b9543120/extensions/json-language-features/package.json#L97), so i just do the same. thanks.